emrun.py

```diff
@@ -479,6 +479,7 @@ def serve_forever(self, timeout=0.5):
       now = tick()
       # Did user close browser?
       if not emrun_options.no_browser and not is_browser_process_alive():
+        logi('Browser process is not alive. Quitting web server.')
```

Comment:
Emrun is not necessarily closing down at this point. See `--serve_after_close` option. To log properly, it should be on the line below after `self.is_running = False`.

Perhaps set the message to "Browser process exited. Quitting web server. (Pass --serve_after_close or --no_browser to keep hosting the page after browser is closed)" to communicate the intent?